templates/_common.tpl

```diff
@@ -94,34 +93,25 @@ volumeMounts:
   coder-managed services.
 */}}
 {{- define "coder.serviceTolerations" }}
-{{- if ne (include "movedValue" (dict "Values" .Values "Key" "services.tolerations")) "" }}
-tolerations:
-{{ include "movedValue" (dict "Values" .Values "Key" "services.tolerations") }}
+{{- if .Values.services.tolerations }}
+tolerations: {{ .Values.services.tolerations }}
 {{- end }}
 {{- end }}
 {{/*
   coder.accessURL is a URL for accessing the coderd.
 */}}
 {{- define "coder.accessURL" }}
-{{- if .Values.cemanager }}
-{{- if ne (merge .Values dict | dig "cemanager" "accessURL" "") "" }}
-{{- .Values.cemanager.accessURL -}}
-{{- else -}}
-    http://cemanager.{{ .Release.Namespace }}{{ include "movedValue" (dict "Values" .Values "Key" "services.clusterDomainSuffix") }}:8080
-{{- end }}
-{{- else -}}
-{{- if ne (merge .Values dict | dig "coderd" "accessURL" "") "" }}
+{{- if .Values.coderd.accessURL "" }}
```

Comment:
```suggestion
{{- if ne .Values.coderd.accessURL "" }}
```

Reply:
I guess this could be `{{- if .Values.coderd.accessURL }}` since the empty value is false? not really sure (and that value is missing from the default values.yaml :thinking:)

Reply:
yeah that should work